saml2aws.go

```diff
@@ -40,8 +40,8 @@ var MFAsByProvider = ProviderList{
 	"PingNTLM":      []string{"Auto"},        // automatically detects PingID
 	"PingOne":       []string{"Auto"},        // automatically detects PingID
 	"JumpCloud":     []string{"Auto", "TOTP", "WEBAUTHN", "DUO", "PUSH"},
-	"Okta":          []string{"Auto", "PUSH", "DUO", "SMS", "TOTP", "OKTA", "FIDO", "YUBICO TOKEN:HARDWARE", "SYMANTEC"}, // automatically detects DUO, SMS, ToTP, and FIDO
-	"OneLogin":      []string{"Auto", "OLP", "SMS", "TOTP", "YUBIKEY", "DUO TOTP"},                                       // automatically detects OneLogin Protect, SMS and ToTP
+	"Okta":          []string{"Auto", "PUSH", "DUO", "SMS", "EMAIL", "TOTP", "OKTA", "FIDO", "YUBICO TOKEN:HARDWARE", "SYMANTEC"}, // automatically detects DUO, SMS, ToTP, and FIDO
+	"OneLogin":      []string{"Auto", "OLP", "SMS", "TOTP", "YUBIKEY", "DUO TOTP"},                                                // automatically detects OneLogin Protect, SMS and ToTP
```

Comment:
I think you accidentally added space here. Are you using tabs?

Reply:
I don't know why. It's added by linter, not me when I run `make lint-fix`. Accually lint check failed first, so I added the second commit(https://github.com/Versent/saml2aws/pull/1281/commits/51cb6bc827c7aba2c1141a1f65b606d6f8cf5a4d)